euth/ideas/models.py

```diff
@@ -20,6 +22,11 @@ def get_absolute_url(self):
         from django.core.urlresolvers import reverse
         return reverse('idea-detail', args=[str(self.slug)])
 
+    def clean(self):
+        super().clean()
```

Comment:
I thought maybe we will implement clean() in the Item at some time and then we wonder why it does not work.